WordPress/Classes/ViewRelated/Blog/QuickStartCell.swift

```diff
@@ -22,6 +22,16 @@ import UIKit
     }
 
     private enum Metrics {
-        static let margins = UIEdgeInsets(top: 16, left: 8, bottom: 8, right: 8)
+        static func margins(for quickStartType: QuickStartType) -> UIEdgeInsets {
+            switch quickStartType {
+            case .undefined:
+                return .zero
```

Comment:
Undefined here means this quick start started on an older version, which basically means the type is `newSite`. I think we should it treat it here this way.

Reply:
Good catch, thanks! Fixed -- f7ac1ba